docs/react-native.md

```diff
@@ -0,0 +1,32 @@
+# React Native
+
+## Installing React Native Expo
+
+We use [Expo](https://expo.dev/) to help us developing our application. Installing the command line tool so you can run Expo commands.
+
+```
+npm install -g expo-cli
+```
+
+## Running the application
+
+Open `Metro Bundler` (deployment server) for testing and deployment purposes.
+
+```
+cd src/mobile
+expo start
+```
+
+`Metro Bundler` give you the chance to test our application on `Android` simulator, `iOS` simulator or on a real device. Testing on `iOS` require `MacOS` and [Xcode](https://developer.apple.com/xcode/). On the other hand, [Android Studio](https://developer.android.com/studio?gclid=Cj0KCQiAuvOPBhDXARIsAKzLQ8HSP0n1tojsRjsEZhT2cWmUrps8u4oND0eAONViZsf9rqMk2gFpc_waAtJOEALw_wcB&gclsrc=aw.ds) providing virtual `Android` machine for testing purpose. In `Metro Bundler`, you can choose to generate `QR` code with the `tunnel` connection to have the app running on your physical device
```

Comment:
There are a few typos:

* `Metro Bundler` **gives** you...
* our application on **an** `Android` simulator...
* Testing on `iOS` **requires** `macOS`...
* Android Studio **provides** **a** virtual `Android` machine for testing **purposes**...

There is also a missing period on the last sentence of this line.

Could you also break this line into several lines? I know that it doesn't matter that much when showing it, but you are editing it, the line is some long.